README.md

```diff
@@ -1 +1,118 @@
-# Agorex - Cooperative Decentralized Exchange
\ No newline at end of file
+# Agorex - Cooperative Decentralized Exchange
+Our official mission statement is currently pending. 
+
+This statment, along with the goals, [future](#road-map), and [members](#members) of Agorex-io are to be determined by the Agorex-io cooperative.  
+
+- [Name](#name)
+- [Membership](#membership)
+- [Road-map](#road-map)
+- [Members](#members)
+- [Social Links](#social-links)
+
+## Name
+Agorex, the Cooperative Decentralized Exchange!
+
+So, what exactly does "Agorex" mean?
+
+Agorex is a combination of two words: Agora & Exchange.
+
+An "Agora" was a "public, open space used for assemblies and markets" in ancient Greece.
+
+As an added bonus "Agora" also means "Now" in Portuguese!
+
+We believe this name represents our core values of providing a public, open, honest, community-run decentralized exchange.
+
+
+## Membership
+If you're interested in contributing to Agorex-io's future, please join our [Discord server](https://discord.gg/GVuj6UY). 
```

Comment:
I think we should focus on "Contribute" rather than "Membership" here. We should strive to answer two questions: _Why_ should you contribute? and _How_ can one contribute? Why might be addressed with a little about what we who we are and what we **believe**:

> We are a group who have worked on a decentralized cryptocurrency exchange before. We want the community, like the technology, to also be decentralized. So we are building a community-run decentralized cryptocurrency exchange that puts the community in charge. To see how we put the community in charge of the direction of the exchange, see our [proposal process](https://github.com/Agorex-io/Proposals). We also plan to set up a cooperative organization where contributors who want to make a more significant contribution can literally operate and manage the exchange on behalf of the community. 

If you want to be part of a community that empowers you to influence the direction of the community and build cryptocurrency technology of a decentralized exchange that is public, open, honest, then come and contribute to our community-run exchange.

How can you contribute:
> You can contribute by joining our discord and offering your ideas, insights, and advice on how to build an public, open, honest community around a decentralized exchange. You can contribute with your insights on all aspects of a community-run decentralized cryptocurrency exchange. Including technology, business/legal, economics, community building, documentation. And of course, we ❤️ pull requests from designers and developers too!